hpx/parallel/segmented_algorithms/detail/scan.hpp

```diff
@@ -637,14 +651,14 @@ namespace hpx { namespace parallel { inline namespace v1
                     hpx::dataflow(
                         policy.executor(),
                         hpx::util::unwrapped(
-                            [=, &op](T last_value, T)
+                            [=, &op, &conv](T last_value, T)
```

Comment:
The return of segmented_scan_void function object is hpx::util::unused. SHould this be the return of he lambda or should it be blank for void?